test/common/stats/thread_local_store_test.cc

```diff
@@ -101,7 +172,7 @@ TEST_F(StatsThreadLocalStoreTest, NoTls) {
   EXPECT_EQ(&g1, &store_->gauge("g1"));
 
   Histogram& h1 = store_->histogram("h1");
-  EXPECT_EQ(&h1, &store_->histogram("h1"));
+  // EXPECT_EQ(&h1, &store_->histogram("h1"));
```

Comment:
If you are going to leave this in and commented out, please add commentary indicating why.